build.csx

```diff
@@ -286,7 +286,7 @@ class DotnetTestSuite : TestSuite
     public override void Execute()
     {
         var xml = Path.GetFullPath(Path.Combine(testsDirectory, Path.GetFileName(this.TestDirectory) + ".TestResults.xml"));
-        Cmd(this.TestDirectory, "dotnet", $"xunit -configuration Release -nologo -notrait \"explicit=yes\" -xml {xml}");
+        Cmd(this.TestDirectory, "dotnet", $"xunit -configuration Release -nologo -notrait \"explicit=yes\" -xml {xml} -nobuild");

```

Comment:
I thought about this late, but the command might be easier to read if the shorter options come first. Maybe before "-notrait"? This is _such_ a minor deal.
I wouldn't've mentioned it if I wasn't already commenting on the outputDirectory issue.

Oh. While I'm here, I see that there are no quotes around `{xml}`, which might cause us problems if a user has spaces in their path. But this is technically a separate issue.